packages/palette/src/elements/Dialog/Dialog.story.tsx

```diff
@@ -0,0 +1,59 @@
+import { storiesOf } from "@storybook/react"
+import React from "react"
+import { Dialog } from "./Dialog"
+
+storiesOf("Components/Dialog", module)
+  .add(
+    "Minimal",
+    () => {
+      return (
+        <Dialog
+          show
+          title="Here is a dialog which is modal"
+          primaryCta={{
+            action: () => ({}),
+            text: "Continue",
+          }}
+        />
+      )
+    },
+    { chromatic: { delay: 500, diffThreshold: 0.2 } }
+  )
+  .add(
+    "With detail",
+    () => {
+      return (
+        <Dialog
+          show
```

Comment:
`show` shouldn't be required, but rather default to true with the option to set it to false